.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
This is amazing! Thank you for the PR! Just 1 request here, can you exclude the `.idea` dir from the PR? I think that it is not needed.